site/cypress/e2e/Cypress-Feature/polls.spec.js

```diff
@@ -119,6 +119,7 @@ describe('Test cases revolving around polls functionality', () => {
         cy.contains('Poll 1').siblings().last().click();
         // make sure all the page elements are there
         cy.get('.content > h1').contains('Poll 1');
+        cy.get('[data-cy="timer"]').contains('Poll Ended');
```

Comment:
```suggestion
        cy.get('[data-testid="timer"]').contains('Poll Ended');
```
Same with other timers